precise.template.spec

```diff
@@ -29,11 +30,13 @@ if train_libs:
     ]
     hidden_imports += ['h5py']
 
+datas = collect_data_files('astor', subdir=None, include_py_files=False)
```

Comment:
What does this return? A list of data files?

Reply:
The astor lib uses an internal file called VERSION.l, which pyinstaller does not include by default.

Pyinstaller does allow you to specify data files which should be bundled along.

See
https://pyinstaller.readthedocs.io/en/stable/spec-files.html#adding-data-files

The function returns a list of tuples where the first is a file name and the second is the folder to insert it to.

Here it will collect all yhe non python files in the root directory of astor.

If that's preferable I think I can specify the exact file instead as shown here:
https://pyinstaller.readthedocs.io/en/stable/spec-files.html#using-data-files-from-a-module